edelphi/src/main/webapp/_scripts/gui/orderplanblockcontroller.js

```diff
@@ -0,0 +1,27 @@
+/*global CONTEXTPATH,BlockController,addBlockController,startLoadingOperation */
+(function() {
+  'use strict';
+  
+  var PlanEditorBlockController = Class.create(BlockController, {
+    initialize: function ($super) {
+      $super();
+      
+      this._orderButtonClickListener = this._onOrderButtonClick.bindAsEventListener(this);
+    },
+    setup: function ($super) {
+      $super($('orderPlanDetailsBlockContent'));
+      this._orderButton = this.getBlockElement().down('input[name="order"]');
+      Event.observe(this._orderButton, "click", this._orderButtonClickListener);
+    },
+    deinitialize: function ($super) {
+      Event.stopObserving(this._orderButton, "click", this._orderButtonClickListener);
+      $super();
+    },
+    _onOrderButtonClick: function () {
+      startLoadingOperation("orderplan.block.preparingOrder");
+    }
+  });
+
+  addBlockController(new PlanEditorBlockController());
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') "addBlockController" does not exist. Change its name or declare it so that its usage doesn't result in a "ReferenceError". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS3827)